packages/gatsby/src/services/initialize.ts

```diff
@@ -314,7 +314,8 @@ export async function initialize({
 
   const state = store.getState()
 
-  const hashes: any = await Promise.all([
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  const hashes: Array<any> = await Promise.all([
```

Comment:
I've changed this in my `feat/ts-aot` branch to this: https://github.com/gatsbyjs/gatsby/blob/93443083259a5b54940393f6cf3f49a8f1abe3a1/packages/gatsby/src/services/initialize.ts#L319-L339

Doesn't block it, just FYI how I changed stuff to make it work